lib/features/family/features/admin_fee/presentation/widgets/admin_fee_text_layout.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:givt_app/features/family/features/admin_fee/presentation/models/admin_fee_uimodel.dart';
+
+class AdminFeeTextLayout extends StatelessWidget {
+  const AdminFeeTextLayout({
+    required this.uiModel,
+    super.key,
+    this.themeData,
+    this.isMonthly = false,
+    this.isMultipleChildren = false,
+    this.textStyle,
+  });
+
+  final AdminFeeUIModel uiModel;
+  final ThemeData? themeData;
+  final TextStyle? textStyle;
+  final bool isMonthly;
+  final bool isMultipleChildren;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = themeData ?? Theme.of(context);
+    final perchild = isMultipleChildren ? ' per child' : '';
+    final feeString = uiModel.fee.toStringAsFixed(2);
+    final monthlyText = 'Admin fee of \$$feeString applies$perchild monthly';
+    final nonMonthlyText = 'An admin fee of \$$feeString will be added';
```

Comment:
Something we need to decide as a team. I said many times I want dumb widgets and to clarify, dumb when it comes to "business" logic. But I think the widget is allowed to be smart about "presentation" logic, i.e. what and how things should look like. Do you guys agree/ disagree?